src/firebolt/async_db/cursor.py

```diff
@@ -550,7 +558,7 @@ async def _api_request(
             url=f"/{path}",
             method="POST",
             params={
-                "database": self.connection.database,
+                "database": self.parameters["database"],
```

Comment:
Do we still need to add database separately? Isn't it a part of self.parameters?

Reply:
You're right. Somehow this _api_request was different from the others.